components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -88,12 +92,11 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
   if (self) {
     if ([aDecoder containsValueForKey:MDCMultilineTextFieldTextViewKey]) {
       _textView = [aDecoder decodeObjectForKey:MDCMultilineTextFieldTextViewKey];
-      [self setupTextView];
-    } else if (!_textView) {
-      // It's expected that this will be created by the fundament above. This is a catch.
-      _textView = [[UITextView alloc] initWithFrame:CGRectZero];
-      [self setupTextView];
     }
+    if (!_textView) {
+      _textView = [[MDCIntrinsicHeightTextView alloc] initWithFrame:CGRectZero];
+    }
+    [self setupTextView];
```

Comment:
Do we want to replace existing UITextViews with IntrinsicHeight versions?
if (!_textView || ![_textView isKindOf:[MDCIntrinsicHeightTextView class]]) {
  _textView = [[MDCIntrinsicHeightTextView alloc] initWithFrame:CGRectZero];
}

Reply:
setupTextView will overwrite a bunch of properties that may already be encoded.  Is this acceptable?  If so, consider adding a comment that mentioning this and that it is intentional.

Reply:
Added the comment.